scipy/signal/tests/test_spectral.py

```diff
@@ -871,6 +872,20 @@ def test_phase_shifted_input(self):
         assert_allclose(f, f1)
         assert_allclose(C, C1)
 
+class TestCyclicSd:
+    def test_identical_input(self):
+        x = np.random.randn(50)
```

Comment:
We need to use a seeded PRNG to generate random data so it's the same every time. `randn()` is a not-preferred alias; we use `standard_normal()` these days.

```suggestion
        rs = np.random.RandomState(526057773)
        x = rs.standard_normal(50)
```